src/components/core/geoChart/geoChartDataLayers.ts

```diff
@@ -46,6 +46,7 @@ function createPushpinSizeOptions(geoData: IGeoData): mapboxgl.Expression | numb
     // The mouseenter event uses queryRenderedFeatures to determine when the mouse is touching a feature
     // And queryRenderedFeatures is not supporting nested object in expression
     // https://github.com/mapbox/mapbox-gl-js/issues/7194
+
```

Comment:
We should remove this line, this explanation aims to `["step", ["get", "pushpinRadius"], DEFAULT_PUSHPIN_SIZE_VALUE]`

Reply:
fixed